### PR TITLE
Pinning to releases and other minor updates

### PR DESCRIPTION
**Pull request by jlstevens**
Updating the dev pins to use new releases as they appear. Updated:

- [x] panel
- [x] holoviews
- [x] hvplot

Only one left is hvplot which is particularly important as that is where the `.interactive` API comes from!

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
 .. raw:: html
-
     <img src="assets/holoviz-logo-stacked.svg" height="150px">
+
+High-level tools to simplify visualization in Python
+----------------------------------------------------
+
+.. raw:: html
+
     <h1>High-level tools to simplify visualization in Python.</h1>
```
Comment (jbednar) [inline doc/index.rst]: Seems to have the same title twice, in two different formats?

Comment (jlstevens) [reply doc/index.rst]: Oops, thought I removed that. Fixed in the next commit.

**Comment by jlstevens** [comment]
@philippjfr @jbednar any idea why appveyor is still being listed despite the yaml files being removed? Maybe this repo needs to be removed from the appveyor web interface somewhere?

**Comment by philippjfr** [comment]
Just deleted it.

**Comment by jlstevens** [comment]
Still see appveyor on this PR but maybe it will be gone after this one is merged.

Edit: Gone now.

**Comment by philippjfr** [comment]
Should be gone the next time you push anything.

**Comment by jlstevens** [comment]
@jbednar @philippjfr Merge? Unless we want to wait for the hvplot release?

**Comment by jbednar** [comment]
Seems like it should wait on the hvplot release, since none of this is fixing an urgent problem.

**Comment by jlstevens** [comment]
I've checked the notebooks still work, updated the lock and the the project on anaconda.org. Ready to merge (assuming the tests go green).

**Comment by jlstevens** [comment]
The linux and OSX builds completed fine but the windows builds seem to time out while running the notebooks for some reason. I would still merge as I think this is an issue with the tests and not the project.

**Comment by jlstevens** [comment]
@ablythed confirmed things are working on windows so this is a testing issue out of the scope of this PR. Merging.

**Pull request merged**